Builds/containers/packaging/dpkg/debian/rules

```diff
@@ -38,12 +51,14 @@ override_dh_auto_configure:
 		-B bld/rippled
 
 	conan install . \
+		--profile gcc \
 		--install-folder bld/rippled-reporting \
 		--build missing \
 		--build boost \
 		--build sqlite3 \
 		--build libuv \
 		--settings build_type=Release \
+		--settings compiler.cppstd=17 \
```

Comment:
We can't build non-reporting mode _without_ C++20, and we can't build reporting mode _with_ C++20? Is that right?

Reply:
It seems to be the case. IIRC, there was an issue building Cassandra.

Reply:
Yeah, that's the issue I'm having:

> C++20 changes std::memory_order from an `enum` to an `enum class`. Datastax supplies a C++ driver for Cassandra, a dependency of reporting mode. it [defines](https://github.com/datastax/cpp-driver/blob/84389657d4504604c5d05b87769ce3ce983a2296/src/atomic/atomic_std.hpp#L24-L31) an `enum MemoryOrder` whose values are copied from `std::memory_order`, but using values from an `enum class` is not allowed, causing a compile error: https://godbolt.org/z/dG943nvT7